README.MD

```diff
@@ -113,10 +113,12 @@ _This is the minimal required configuration. Take a look to the [Account Kit doc
 <string name="ak_client_token">YOUR_CLIENT_TOKEN</string>
 ```
 
-2.  In `android/app/src/main/AndroidManifest.xml`
+2.  In `android/app/src/main/AndroidManifest.xml` make sure that you have added following lines
 
 ```xml
 ...
+<uses-permission android:name="android.permission.READ_SMS" />
```

Comment:
Thanks for the PR! I would make only a small change. Maybe you can put a more specific step between the current 2 and 3 saying something like:

2. ...
3. If you want to automatically detect your phone number and/or autofill the received SMS add the following permissions to the `android/app/src/main/AndroidManifest.xml` file:

  ```xml
  <uses-permission android:name="android.permission.READ_SMS" />
  <uses-permission android:name="android.permission. RECEIVE_SMS" />
  ```
4. ...

Thoughts?

Reply:
ok